PCL_ConnectedComponents/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:latest
+
+ARG RELISE_VERSION=1.12.1
+
+RUN apt-get update && \
+    apt-get install -y \
+      gcc \
+      mono-mcs \
+      libboost-all-dev \
+      libeigen3-dev \
+      libflann-dev \
+      libvtk6-dev \
+      python3-vtk7 \
+      libusb-1.0-0-dev \
+      wget \
+      cmake \
+      python3 \
+      python3-pip \
+      libglu1-mesa-dev freeglut3-dev mesa-common-dev
+
+
+RUN pip3 install open3d
+
+ADD . /app/build
+WORKDIR /app/build
+
+RUN wget -qO- https://github.com/PointCloudLibrary/pcl/archive/refs/tags/pcl-${RELISE_VERSION}.tar.gz | tar xz \
```

Comment:
It will be a good idea to create common docker image for pcl library and use as base image for dockers of both pcl-based algos